content/scout/integrations/registry/artifactory.md

```diff
@@ -179,14 +179,12 @@ line flag.
 
 You can view the image analysis results in the Docker Scout Dashboard.
 
-1. Go to [Docker Scout Dashboard](https://scout.docker.com).
-2. Sign in using your Docker ID.
+1. Go to [Images page](https://scout.docker.com) in the Docker Scout Dashboard.
```

Comment:
Question: This link takes me to the **Overview** page - not a big deal but is this step consolidating two steps? (Open the dashboard, then select **Images** in the right-hand nav)

Reply:
Yeah meant to point it to the images page, updated the url